optuna/pruners/hyperband.py

```diff
@@ -80,8 +80,14 @@ class HyperbandPruner(BasePruner):
             The number of :class:`~optuna.pruners.SuccessiveHalvingPruner`\\ s (brackets).
             Defaults to :math:`4`.
         min_early_stopping_rate_low:
+
+            .. deprecated:: 1.4.0
+                This argument will be removed from :class:~optuna.pruners.HyperbandPruner. The
+                 minimum value of the minimum early-stopping rate are automatically determined
+                 based on ``min_resource`` and ``reduction_factor``.
```

Comment:
Isn't it fixed to 0, and not based on `min_resource` or `reduction_factor`?

Reply:
Sorry for confusion. This explanations should be removed.